contracts/axelar-gateway/src/contract.rs

```diff
@@ -47,7 +48,12 @@ impl AxelarGateway {
         Ok(())
     }
 
-    /// Call a contract on another chain with the given payload. The destination address can validate the contract call on the destination gateway.
+    /// Sends a message to the specified destination chain and contarct address with a given payload.
+    ///
+    /// This function is the entry point for general message passing between chains.
+    ///
+    /// A registered chain name on Axelar must be used for `destination_chain`. `payload` usually represents
+    /// an encoded function call with arguments.
```

Comment:
```suggestion
    /// A registered chain name on Axelar must be used for `destination_chain`.
```
does this line come from EVM? it's not just an encoded function call. `execute` method of the dApp gets called everytime

Reply:
yup it does: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/9847262542dd2279d3a14972f35cf85900657ca9/contracts/gateway/BaseAmplifierGateway.sol#L37C1-L37C135

Reply:
I see, can you fix it there as well?

Reply:
done: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/pull/196